lib/http_adapter/angular.js

```diff
@@ -8,8 +8,13 @@
     function AngularAdapter($http, $window) {
         this._$http = $http;
         this._$window = $window;
+        this._timeout = 10000;
     }
 
+    AngularAdapter.prototype.setTimeout = function(timeout) {
```

Comment:
Good idea. If there are any extra options we'll add in the future, this seems like a good path.